src/material-examples/example-module.ts

```diff
@@ -129,696 +123,485 @@ import {ToolbarOverviewExample} from './toolbar-overview/toolbar-overview-exampl
 import {TooltipOverviewExample} from './tooltip-overview/tooltip-overview-example';
 import {TooltipPositionExample} from './tooltip-position/tooltip-position-example';
 
-export const EXAMPLE_COMPONENTS = {
+export interface LiveExample {
+  title: string;
+  component: any;
+  additionalFiles?: string[];
+  selectorName?: string;
+}
+
+export const EXAMPLE_COMPONENTS: {[key: string]: LiveExample} = {
```

Comment:
This file is generated- did you update the script that writes it?

Reply:
AFAIK that’s the example-data.ts. It already had all the null checks.

Reply:
`gulp build-examples-module` writes this file

Reply:
Oh I see, I must've missed the warning because of all the imports. I've updated the generation script now.